momentum/4/modules/msg-gen.md

```diff
@@ -154,6 +154,32 @@ Maximum allowed time in the future for which a transmission can be scheduled. Ti
 
 </dl>
 
+### <a name="modules.msg_gen.scheduled_send"></a> Scheduled Send
```

Comment:
Instead of a new header, I think we should have the formatting here match the above msg_gen options.

So for example, the format for the "scheduled_send" option would be identical to how the "start_time_max_schedule_interval" option is described above.